components/org.wso2.carbon.identity.oauth2.grant.rest.core/src/main/java/org/wso2/carbon/identity/oauth2/grant/rest/core/RestAuthenticationServiceImpl.java

```diff
@@ -1073,4 +1087,95 @@ private String getFederatedIdpChannelName(String authenticatorName, RestAuthenti
         return null;
     }
 
+    /**
+     * This method is used to get user's channel identifier attribute.
+     *
+     * @param userId                          User's user id.
+     * @param userTenantId                    User's tenant id.
+     * @param requestedClaimURI               Required channel Identifier's claim URI.
+     * @throws AuthenticationServerException  Returns an AuthenticationException.
+     * @return String                         User's chanel Identifier's value.
+     */
+    private String getUserVerificationChannel(String userId, int userTenantId, String requestedClaimURI)
+            throws AuthenticationServerException {
+
+        AbstractUserStoreManager userStoreManager;
+        Map<String, String> userAttributes;
+        try {
+            userStoreManager = (AbstractUserStoreManager) AuthenticationServiceDataHolder.getInstance()
+                    .getRealmService().getTenantUserRealm(userTenantId).getUserStoreManager();
+            userAttributes = userStoreManager.getUserWithID(userId, new String[]{requestedClaimURI},
+                    UserCoreConstants.DEFAULT_PROFILE).getAttributes();
+
+            if(LOG.isDebugEnabled()) {
+                LOG.debug("Verifier claim value resolved for the user id : " + userId);
```

Comment:
let's add the resolved claim value as well for the verification channel in the log

Reply:
Fixed with 28d24d24dd8ae8ac604217dff5d2b45cd3140957